L1Trigger/TrackFindingTracklet/test/L1TrackNtupleMaker_cfg.py

```diff
@@ -40,27 +41,29 @@
 process.MessageLogger.Tracklet = dict(limit = -1)
 process.MessageLogger.TrackTriggerHPH = dict(limit = -1)
 
+
+print("using geometry " + GEOMETRY + " (tilted)")
+process.load('Configuration.Geometry.GeometryExtendedRun4' + GEOMETRY + 'Reco_cff')
+process.load('Configuration.Geometry.GeometryExtendedRun4' + GEOMETRY +'_cff')
+
```

Comment:
Move these lines from https://github.com/cms-sw/cmssw/blob/8425d08b0f4164a98c20a56168487466f474b1ba/L1Trigger/TrackFindingTracklet/test/L1TrackNtupleMaker_cfg.py#L58-L59 to here:
```suggestion
process.load('Configuration.StandardSequences.EndOfProcess_cff')
process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```
As the load of the GlobalTag needs to before you then modifying the `process.GlobalTag`.